src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -0,0 +1,209 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <vector>
+
+#include "Domain/Creators/DomainCreator.hpp"  // IWYU pragma: keep
+#include "Domain/Domain.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace domain {
+namespace creators {
+
+/*!
+ * \ingroup DomainCreatorsGroup
+ *
+ * \brief A Domain for testing purposes.
```

Comment:
This looks like it still says `A domain for testing purposes`, previous comments suggested it to be changed to `A general domain for two compact objects`